Platon/backend/app/auth_system/views.py

```diff
@@ -346,63 +346,61 @@ def put(requester_id, self):
         # If yes, starts processing the data.
         # If not, an error is raised.
         if form.validate():
-            # Checks whether there is any field data sent.
-            # If not, returns the message accordingly.
-            # If yes, starts acting upon the request.
-            if not any(form.data.values()):
-                return make_response(jsonify({"message" : "Server has received the request but there was no information to be updated."}), 202)
+            # Tries to connect to the database.
+            # If it fails, an error is raised.
+            try:
+                existing_user = User.query.filter_by(id=requester_id)
+            except:
+                return make_response(jsonify({"error" : "The server is not connected to the database."}), 500)
             else:
```

Comment:
why did you use else statement here? Because there is no if statement above

Reply:
@cakmadam98 Try-except blocks can have an "else" part too. "Else" in this context means "try, if no exception is raised, continue here."